guidelines/index.html

```diff
@@ -1282,22 +1285,22 @@ <h4 id="x1-3-3-sensory-characteristics"><span class="secno">達成基準 1.3.3 <
 </section>
 
 
-                <section class="sc new" id="orientation">
+                <section class="sc" id="orientation">
    					
    <h4 id="x1-3-4-orientation"><span class="secno">達成基準 1.3.4 </span>表示の向き<span class="permalink"><a href="#orientation" aria-label="Permalink for 1.3.4 Orientation" title="Permalink for 1.3.4 Orientation"><span>§</span></a></span></h4>
    					
    <div class="doclinks"><a href="https://www.w3.org/WAI/WCAG21/Understanding/orientation.html">Understanding Orientation</a> <span class="screenreader">|</span> <br><a href="https://www.w3.org/WAI/WCAG21/quickref/#orientation">How to Meet Orientation</a></div><p class="conformance-level">(レベル AA)</p>
    					
    
    					
-  <p>コンテンツは、その表示及び操作を、縦向き (portrait) 又は横向き (landscape) などの単一の向きに制限しない。ただし、その表示の向きが<a href="#dfn-essential" class="internalDFN" data-link-type="dfn">必要不可欠</a>な場合は例外とする。</p>
+  <p>コンテンツは、その表示及び操作を、縦向き (portrait) 又は横向き (landscape) のような単一の表示の向きに制限しない。ただし、特定の表示の向きが<a href="#dfn-essential" class="internalDFN" data-link-type="dfn">必要不可欠</a>な場合は除く。</p>
    				
-   <div class="note" id="issue-container-generatedID-0"><div role="heading" class="note-title marker" id="h-note-0" aria-level="5"><span>注記</span></div><p class="">単一の表示の向きが必要不可欠な例としては、銀行の小切手、ピアノのアプリケーション、プロジェクタやテレビ向けのスライド、バイナリディスプレイ表示が適用されないバーチャルリアリティのコンテンツなどが挙げられる。</p></div>
+   <div class="note" id="issue-container-generatedID-0"><div role="heading" class="note-title marker" id="h-note-0" aria-level="5"><span>注記</span></div><p class="">特定の表示の向きが必要不可欠な例としては、銀行の小切手、ピアノのアプリケーション、プロジェクター又はテレビ向けのスライド、コンテンツが必ずしも横向き (landscape) 又は縦向き (portrait) の表示の向きに制限されないバーチャルリアリティのコンテンツが挙げられる。</p></div>
```

Comment:
単なる感想として、バーチャルリアリティのコンテンツは「特定の表示の向きが必要不可欠な例」とは言えない (むしろ逆で、縦or横のいずれかの向きに制限できない) と思うので、そこは少し気になりました。
原文は  may be essential なので、may be を訳出すると違和感が薄まったりしますかね?
どうでしょう……?

```suggestion
   <div class="note" id="issue-container-generatedID-0"><div role="heading" class="note-title marker" id="h-note-0" aria-level="5"><span>注記</span></div><p class="">特定の表示の向きが必要不可欠となり得る例としては、銀行の小切手、ピアノのアプリケーション、プロジェクター又はテレビ向けのスライド、コンテンツが必ずしも横向き (landscape) 又は縦向き (portrait) の表示の向きに制限されないバーチャルリアリティのコンテンツが挙げられる。</p></div>
```